docs/contributors/code/release.md

```diff
@@ -130,15 +130,33 @@ If you decide that the fixes deserve another release candidate before the stable
 
 ### Creating Point Releases
 
-Occasionally it's necessary to create a point release (i.e. X.Y._Z_) of the Plugin.
+Occasionally it's necessary to create a point release (i.e. X.Y._Z_) of the Plugin. This is most commonly to push fixes for regressions and/or bugs.
 
-The method for doing this is nearly identical to the main Plugin release process (see above). In the `Run workflow` control specify `stable` and, _if_ the previous version was stable (`x.y.z`), the workflow will release a point release, with z incremented (`x.y.(z+1)`) as necessary.
+As you proceed with the following process, it's worth bearing in mind that such point releases are not created as branches in their own right (e.g. [`release/12.5.0`](https://github.com/WordPress/gutenberg/tree/release/12.5)) but are simply [tags](https://github.com/WordPress/gutenberg/releases/tag/v12.5.1).
```

Comment:
No need to point to a different thing here, as the `12.5.0` branch do not exists.
```suggestion
As you proceed with the following process, it's worth bearing in mind that such point releases are not created as branches in their own right (e.g. `release/12.5.0`) but are simply [tags](https://github.com/WordPress/gutenberg/releases/tag/v12.5.1).
```